install.sh

```diff
@@ -65,12 +70,18 @@ Repeat the password: "
   local repeat_retry="
 Passwords do not match! Try again.
 "
-
+local validation_prompt="
+Password must be at least 8 characters long !.
+"
   password=$(get_password "${prompt}")
+  #validate the password entered 
+  until validate_password "${password}"; do
+  password=$(create_password "${validation_prompt}")
```

Comment:
Recursion is nice. In this case, it results in the user being asked to repeat their password `r` times, where `r` is the recursion depth (if they always get the repeated password correct; otherwise it's more times).